lintrunner_adapters/adapters/refurb_linter.py

```diff
@@ -0,0 +1,164 @@
+
+    >>> t(r"main.py:3:17 [FURB109]: Use `in (x, y, z)` instead of `in [x, y, z]`")
+    ... # doctest: +NORMALIZE_WHITESPACE
+    {'file': 'main.py', 'line': '3', 'column': '17', 'code': 'FURB109',
+     'message': 'Use `in (x, y, z)` instead of `in [x, y, z]`'}
+
+    >>> t(r"main.py:4:5 [FURB101]: Use `y = Path(x).read_text()` instead of `with open(x, ...) as f: y = f.read()`")
+    ... # doctest: +NORMALIZE_WHITESPACE
+    {'file': 'main.py', 'line': '4', 'column': '5', 'code': 'FURB101',
+     'message': 'Use `y = Path(x).read_text()` instead of `with open(x, ...) as f: y = f.read()`'}
+    """
+    pass
+
+
+def format_lint_message(message: str, code: str, show_disable: bool) -> str:
+    formatted = f"{message}\n"
+    if show_disable:
+        formatted += dedent(
+            f"""
+            To disable, use
+            [tool.refurb]
+            ignore = [{code}]
+            or
+            disable = [{code}]
+            """
+        )
+    return formatted
+
+
+def check_files(
+    filenames: list[str],
+    severities: dict[str, LintSeverity],
+    *,
+    config_file: str,
+    retries: int,
+    show_disable: bool,
+) -> list[LintMessage]:
+    try:
+        proc = run_command(
+            [sys.executable, "-mrefurb", "--config-file", config_file] + filenames,
+            retries=retries,
+        )
+    except OSError as err:
+        return [
+            LintMessage(
+                path=None,
+                line=None,
+                char=None,
+                code=LINTER_CODE,
+                severity=LintSeverity.ERROR,
+                name="command-failed",
+                original=None,
+                replacement=None,
+                description=(f"Failed due to {err.__class__.__name__}:\n{err}"),
+            )
+        ]
+    stdout = str(proc.stdout, "utf-8").strip()
+    return [
+        LintMessage(
+            path=match["file"],
+            name=match["code"] or "note",
+            description=format_lint_message(
+                match["message"],
+                match["code"],
+                show_disable,
+            ),
+            line=int(match["line"]),
+            char=int(match["column"])
+            if match["column"] is not None and not match["column"].startswith("-")
+            else None,
+            code=LINTER_CODE,
+            severity=severities.get(match["code"], LintSeverity.ERROR),
+            original=None,
+            replacement=None,
+        )
+        for match in RESULTS_RE.finditer(stdout)
+    ]
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        description=f"refurb wrapper linter. Linter code: {LINTER_CODE}",
+        fromfile_prefix_chars="@",
+    )
+    parser.add_argument(
+        "--config-file",
+        required=True,
+        help="path to pyproject.toml config file",
+    )
+    parser.add_argument(
+        "--severity",
+        action="append",
+        help="map code to severity (e.g. `FURB109:advice`)",
```

Comment:
What should users input if they wanted to map more than one codes? Would be very nice to show in the example

Reply:
Use multiple --severity arg